guava/src/com/google/common/collect/ImmutableCollection.java

```diff
@@ -163,7 +163,7 @@
 @SuppressWarnings("serial") // we're overriding default serialization
 // TODO(kevinb): I think we should push everything down to "BaseImmutableCollection" or something,
 // just to do everything we can to emphasize the "practically an interface" nature of this class.
-public abstract class ImmutableCollection<E> extends AbstractCollection<E> implements Serializable {
+public abstract class ImmutableCollection<E extends @Nullable Object> extends AbstractCollection<E> implements Serializable {
```

Comment:
The Google stubs have `extends @NonNull Object` here, and the class Javadoc says the class will never contain a null element.